data/tribes/initialization/frisians/units.lua

```diff
@@ -894,183 +894,253 @@ tribes:new_tribe {
       {
          name = "berry_bush_blueberry_tiny",
          helptexts = {
+            -- TRANSLATORS: Helptext for a frisian immovable: One of the berry bushes
+            purpose = _("This berry bush has just been planted.")
          }
       },
       {
          name = "berry_bush_blueberry_small",
          helptexts = {
+            -- TRANSLATORS: Helptext for a frisian immovable: One of the berry bushes
+            purpose = _("This berry bush is growing.")
          }
       },
       {
          name = "berry_bush_blueberry_medium",
          helptexts = {
+            -- TRANSLATORS: Helptext for a frisian immovable: One of the berry bushes
+            purpose = _("This berry bush is growing.")
```

Comment:
I just remembered myself of medium bushes being flowering to produce honey. So I suggest to change the helptexts of medium bushes to:
This berry bush is flowering. Honey can be produced from it by a beekeeper.

Reply:
Good idea. I have added this to the medium fields etc. too.